Fix appsettings.json structure

mocsharp opened the pull request:
### Description

Fixes #525 

Fix a regression caused by incorrect structure in the appsettings.json file.

### Status
**Ready**

### Types of changes
<!--- Put an `x` in all the boxes that apply, and remove the not applicable items -->
- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [x] New tests added to cover the changes.
- [x] All tests passed locally.
- [ ] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.
- [ ] User guide updated.
- [ ] I have updated the changelog

mocsharp committed:
Fix appsettings.json structure.

Signed-off-by: Victor Chang <vicchang@nvidia.com>

mocsharp committed:
Update libc6-dev installation in Dockerfile to use wildcard version

Signed-off-by: Victor Chang <vicchang@nvidia.com>

MMelQin commented on Dockerfile:
```diff
@@ -36,7 +36,7 @@ ENV DEBIAN_FRONTEND=noninteractive
  && apt-get update \
  && apt-get install -y --no-install-recommends curl \
- && apt-get install -y libc6-dev=2.35-0ubuntu3.9 \
+ && apt-get install -y libc6-dev=2.35-* \
```
Could move the wildcard to the patch level, e.g. `libc6-dev=2.35-0ubuntu3.*`. The latest is `2.35-0ubuntu3.10` that fixes some security vulnerabilities.

MMelQin reviewed (approved):
Looks good to me, and the nitpick by the copilot can be somewhat addressed by having the wildcard at the patch level (likely to avoid the main line version `3` and get the latest patch `3.*`.

mocsharp committed:
Update libc6-dev version in Dockerfile to specific release

Adjusted the installation command for libc6-dev to use the specific version 2.35-0ubuntu3.* for improved compatibility.

Signed-off-by: Victor Chang <vicchang@nvidia.com>
